src/index.js

```diff
@@ -28,12 +28,12 @@ button.onclick = function processLogin() {
                     .then(function (result) {
                       var destinations = result.travelerDestinations;
                       var trips = result.travelerTrips;
-                      let travelerDashboardData =
-                        traveler_dashboard_helper.prepareTravelerDashboardData(
-                          traveler,
-                          destinations,
-                          trips
-                        );
+                      // eslint-disable-next-line max-len
+                      let travelerDashboardData = traveler_dashboard_helper.prepareTravelerDashboardData(
```

Comment:
shouldnt it be  travlerDashboardHelper instead of traveler_dashobard_helper

I usually only see lowercase with underscore seperating it for filenames.  javascript uses snakeCaseConventions